alphaevolve_gs_engine/examples/run_synthesis_example.py

```
@@ -0,0 +1,198 @@
+        # For instance, make the mock LLM actually generate some valid Rego-like code
+        class CustomMockLLM(MockLLMService):
+            def generate_text(self, prompt: str, max_tokens: int = 1024, temperature: float = 0.7, model:str=None) -> str:
+                # Simple mock Rego generation based on keywords in prompt
+                if "operational_rule" in prompt.lower() and "rego" in prompt.lower():
+                    package_name = "company.access_control" # Default
+                    if "package name" in prompt:
+                        try:
+                            # Attempt to extract package name from prompt (very basic)
+                            import re
+                            match = re.search(r"package named '([^']*)'", prompt)
+                            if match: package_name = match.group(1)
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)